src/client/apps/edit/components/display/components/partner.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box, color, Flex, Serif } from "@artsy/palette"
+import { onChangeArticle } from "client/actions/edit/articleActions"
+import { CharacterLimit } from "client/components/character_limit"
+import { FormLabel } from "client/components/form_label"
+import React from "react"
+import { connect } from "react-redux"
+import styled from "styled-components"
+import { DisplayProps } from "../index"
+import { FillTitlePrompt } from "./magazine"
+const ImageUpload = require("client/apps/edit/components/admin/components/image_upload.coffee")
+
+export const DisplayPartner: React.SFC<DisplayProps> = props => {
+  const { article, onChangeArticleAction } = props
+
+  return (
+    <div>
+      <Intro pt={50}>
```

Comment:
I keep thinking we need to add a `5` entry in the Theme file, though it seems like they deliberately didn't include.